openstack/compute/v2/flavors/doc.go

```diff
@@ -59,5 +59,18 @@ Example to List Flavor Access
 	for _, access := range allAccesses {
 		fmt.Printf("%+v", access)
 	}
+
+Example to Grant Access to a Flavor
+
+	flavorID := "e91758d6-a54a-4778-ad72-0c73a1cb695b"
+
+	accessOpts := flavors.AddAccessOpts{
+		Tenant: "15153a0979884b59b0592248ef947921",
```

Comment:
Since this is the documentation, shouldn't this be an UUID to provide the additional visual prompt as to an appropriate value? I know in the AddAccessOpts it states project/tenant ID, but I think it would be helpful to change here. It's a easy change, but not necessarily enough to hold up the patch on.

Reply:
The rest of the patch looks great.

Reply:
At least on Ocata, the project and user IDs are these kinds of strings rather than UUIDs:

```
Successfully created project ACPTTESTWZC1JOxR with ID 8befa1ba8c224584bb2b7b55c6e35a24
```

 Has this changed post-Ocata?

Reply:
no, you're right. sorry for the noise.

Reply:
It's no problem at all.